doc/user/content/overview/open-cdc.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Materialize Open CDC"
+description: "The Open CDC format allows Materialize to detect when records are duplicated or out of order."
+menu:
+  main:
+    parent: "overview"
+    weight: 4
+---
+
+Change data capture (CDC) tools provide feeds that record any changes to a database. Typically, the feeds are then saved to another platform, like Kafka, for storage or processing.
+
+However, sometimes the stream can have missing or duplicate records, or send records out of order. For example, if a CDC tool crashes while writing a record, it may retry and write the record again, resulting in a duplicate entry.
+
+To prevent this kind of error, Materialize has developed the **Open CDC format**, a format for the updates that CDC tools propagate to other systems.  The Open CDC format is designed especially to prevent record duplication or missequencing by including enough information for a downstream data consumer (like Materialize) to recognize when records are missing, duplicated, or out of order. Each record contains:
+
+- the changes in a record (the updated fields or the deletion or addition of a new record)
+- the timestamp for the change
+- the number of messages used to transmit the record
+
+The timestamp allows Materialize to re-order the events when they're received asynchronously, and the number of messages allows Materialize to determine when there are duplicate updates.
+
+For a technical deep dive on the subject, see our blog post on [Change Data Capture](https://materialize.com/change-data-capture-part-1/).
+
+**Note**: Currently, the Open CDC format is only supported for [Avro Kafka](/sql/create-source/avro-kafka) sources. If you're interested in using it for another source type, let us know in our [Slack workspace](https://materialize.com/s/chat).
+
+## Example Open CDC Schema
+
+You define the Open CDC format in the Avro schema when a source is created.
+
+
+```sql
+  CREATE MATERIALIZED SOURCE name_of_source
+  FROM KAFKA BROKER 'kafka_url:9092' TOPIC 'name_of_kafka_topic'
+  FORMAT AVRO USING SCHEMA '<schema goes here>'
+  ENVELOPE MATERIALIZE
+```
+
+The following schema specifies that records will consist of `id` and `price` fields.
```

Comment:
@umanwizard is it correct that we need the user to spell out all of the update and progress grit in the schema they present to the source? Is it possible to thin that down to just the fields of interest for the user, and then surgery up the rest around it? Sorry if I should know already. :)

Reply:
@umanwizard, I still have this as listing/requiring all the fields in the source in materialize-cdc.md, so please let me know if that should be changed.

Reply:
per Brennan in Slack: "they need to specify the schema of the data, at least. In principle we could take that and jam it into a larger avro schema that handles the progress/updates and such. This is experimental anyway so for sure we can work on the UI! I don't think that needs to block the docs of the current version, though."